tools/run_tests/run_tests.py

```diff
@@ -195,16 +195,22 @@ def __str__(self):
 class NodeLanguage(object):
 
   def __init__(self):
+    self.platform = platform_string()
     self.node_version = '0.12'
 
   def test_specs(self, config, args):
-    return [config.job_spec(['tools/run_tests/run_node.sh', self.node_version],
-                            None,
-                            environ=_FORCE_ENVIRON_FOR_WRAPPERS)]
+    if self.platform == 'windows':
+      return [config.job_spec(['tools\\run_tests\\run_node.bat'], None)]
+    else:
+      return [config.job_spec(['tools/run_tests/run_node.sh', self.node_version],
+                              None,
+                              environ=_FORCE_ENVIRON_FOR_WRAPPERS)]
 
   def pre_build_steps(self):
-    # Default to 1 week cache expiration
-    return [['tools/run_tests/pre_build_node.sh', self.node_version]]
+    if self.platform == 'windows':
```

Comment:
why is not node version required on windows? I thought we are using nvm there as well.